econml/dowhy.py

```diff
@@ -106,30 +110,41 @@ def fit(self, Y, T, X=None, W=None, Z=None, *, outcome_names=None, treatment_nam
         -------
         self
         """
-
-        Y, T, X, W, Z = check_input_arrays(Y, T, X, W, Z)
```

Comment:
Can you add a test that reflects this fix?

Reply:
Done